locale/pt-br/meta.lua

```diff
@@ -523,7 +523,7 @@ math.maxinteger['>5.3']     =
 'Retorna o valor máximo para um inteiro.'
 math.min                    =
 'Retorna o argumento com o valor mínimo de acordo com o operador `<`.'
-math.mixinteger['>5.3']     =
```

Comment:
I think this was a typo, it was making the `locale` script crash.

Reply:
I have resolved it, please pull latest commits.